packages/experimental/TabList/src/Tab/Tab.tsx

```diff
@@ -59,11 +62,15 @@ export const Tab = compose<TabType>({
 
       const { icon, tabKey, ...mergedProps } = mergeProps(tab.props, final, { accessibilityLabel: final.accessibilityLabel || text });
 
+      if (__DEV__ && !text && !icon) {
+        console.warn('A Tab component must render content. Text, an icon, or both should at least be passed in.');
+      }
+
       return (
         <Slots.root {...mergedProps}>
           <Slots.stack>
             {icon && <Slots.icon {...icon} />}
-            <Slots.content>{text}</Slots.content>
+            {text && <Slots.content accessible={false}>{text}</Slots.content>}
```

Comment:
Why is accessible set to false here?

Reply:
So it doesn't get included in the accessibility tree. Text and/or images in teh accessibility tree isn't necessary unless it contains additional information. In most cases, the accessibilityLabel set is enough to describe the tab component. If an accessibilityLabel isn't set, we set it to the value of the text component. We do this for button too.

Reply:
This is so that the Tab items in the UIA tree don't render the text node under the tab node, sort of like what the FURN button does.

Reply:
Makes sense, thanks for the context!